ci/build_libcudacxx.sh

```diff
@@ -17,5 +17,5 @@ cmake -S .. -B ../build \
     -DTHRUST_ENABLE_INSTALL_RULES=ON \
     -G Ninja
 
-LIBCUDACXX_SITE_CONFIG="../build/libcudacxx/test/lit.site.cfg" lit -v --no-progress-bar --no-execute -Dcompute_archs=${GPU_ARCHS} -Dstd="c++${CXX_STANDARD}" ../libcudacxx/.upstream-tests/test
```

Comment:
The build script runs on CPU only machines. So that's why I'm using `--no-execute`.